data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+					decline
+			
+			label limits
+			`	"I mean that they are prevented from reaching their true potential. Once we had many stars and planets to call ours, but since then most of the Hai have decided to stop changing, to stay peaceful with all species, no matter how primitive. They have chosen to decay, to grow soft. The Unfettered are the ones who choose to keep our legacy, our way of life and then to evolve, to grow."`
+			choice
+				`	"What is it exactly you want me to do?"`
+				`	"I am not interested in moving people around."`
+					decline
+			
+			label action
+			`	"Oh it is pretty simple actually, you only need to go to <stopovers>, pick up <bunks> Hai and bring them to this planet, they will find their way from there." You somehow doubt that affirmation, seeing how everything here is in a deprecated state to say the least. You hope that at least the Hai actually know about that fact, or you may actually have to bring them back...`
+			choice
+				`	"I will get to it right away then."`
+					accept
+				`	"I do not want to bring you Hai defectors, sorry."`
+					decline
+	
+	on stopover
+		dialog `The Hai seem happy to get on your ship, now return to <origin> with them.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		clear "Unfettered: offered bring hai"
+		log `Helped some Hai citizens to join the Unfettered.`
+		payment 100000
+		dialog `The Unfettered thank you for helping these "True" Hai find their way home, and give you <payment>. After discussing with them, the Unfettered elder says, "If you want to help us again, the hospitals on Darkcloak are in grave need of resupply." Checking back on the group, you can see they are doing fine, and already finding a place to call home, shared among the others. They may not have much, but they seem not to require anything at all, other than to be here.`
+
+
+
+mission "Unfettered: Bring Medical Supplies Notification"
+	landing
+	name `Need for medical supplies`
+	description `An Unfettered elder told you to come help the hospitals on <destination> that desperately need supplies.`
+	source "Firelode"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Bring Hai: done"
+
+
+
+mission "Unfettered: Bring Medical Supplies"
+	landing
+	name `Bring medicals to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	source "Darkcloak"
+	stopover "Snowfeather"
+	cargo "medicals" 120
+	blocked `You need 120 cargo space available to carry the medicals supplies.`
+	deadline 8
+	to offer
+		has "Unfettered: Bring Hai: done"
+	on offer
+		conversation
+			`As you land, you receive a distress message, originating from one of the hospitals nearby. Staff are asking all who can to immediately bring back medical supplies from <stopovers>.`
```

Comment:
This needs reworking. Ideally include the actual message, and "distress" is probably inappropriate, maybe urgent PSA.